src/modules/getBullsAndCows.js

```diff
@@ -14,6 +14,30 @@
  */
 function getBullsAndCows(userInput, numberToGuess) {
   /* Write your code here */
+  const userInputArr = userInput.toString().split('');
+  const numberToGuessArr = numberToGuess.toString().split('');
+  const result = { bulls: 0, cows: 0 };
+
+  for (let i = 0; i < userInputArr.length; i++) {
+    if (
+      numberToGuessArr.includes(userInputArr[i]) &&
+      numberToGuessArr.indexOf(userInputArr[i]) === i
+    ) {
+      result.bulls += 1;
+    } else if (numberToGuessArr.includes(userInputArr[i])) {
```

Comment:
The condition for checking cows should ensure that the digit is not already counted as a bull. Consider adding a check to ensure the digit is not in the correct position before counting it as a cow.